labgrid/driver/shelldriver.py

```diff
@@ -95,8 +95,9 @@ def _run(self, cmd, *, timeout=30.0, codec="utf-8", decodeerrors="strict"):
         # hide marker from expect
         cmp_command = f'''MARKER='{marker[:4]}''{marker[4:]}' run {shlex.quote(cmd)}'''
         self.console.sendline(cmp_command)
+        # match output by run function, bracketed-paste escape sequence (optionally), prompt
         _, _, match, _ = self.console.expect(
-            rf'{marker}(.*){marker}\s+(\d+)\s+{self.prompt}',
+            rf'{marker}(.*){marker}\s+(\d+)\s+(\x1b\[\?2004h)?{self.prompt}',
```

Comment:
This only matches the escape sequence at a specific location, which seems fragile.

Reply:
Agreed.

Reply:
If (on a given system) this always appears at the start of the prompt, the user could add this to their prompt regex instead.

Reply:
Correct. On the one hand it feels a bit like a work-around, but on the other you could probably argue that this belongs to the prompt somewhat. So I agree that we consider this a valid solution.